webpack/redux/actions/RedHatRepositories/helpers.js

```diff
@@ -31,14 +31,15 @@ const recommendedRepositoriesSatTools = [
   'satellite-client-6-for-rhel-8-x86_64-rpms',
   'rhel-7-server-els-satellite-client-6-rpms',
   'rhel-7-server-els-satellite-6-client-2-rpms',
```

Comment:
Should this rhel-7 client-2 repository also be removed from the list of recommended repositories?

Reply:
Yeah correct. I missed it.